### ISSUE TITLE
help:system clashes with system command group

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#163]
```
What steps will reproduce the problem?
1. Type "help:system"
2. Type "help:camera"
3. Type "help:photo"
4. Type "help:cat"

What is the expected output?
1. For the system commands help to be displayed.
2. For the camera help to be displayed.
3. For the camera help to be displayed.
4. For the system command list to include "camera" and "photo".

What do you see instead?
1. No reply.
2. No reply.
3. No reply.
4. The system command list does not include "camera" or "photo".

Nothing of note in the log.

Maybe because of empty help @ 
http://code.google.com/p/gtalksms/source/browse/src/com/googlecode/gtalksms/cmd/
CameraCmd.java#146 ?
Doesn't seem to explain why help:system doesn't respond, though.

Additional informations:
GTalkSMS version = 2.2.3
Android OS version = 2.2
Phone model = HTC Hero

```

Original issue reported on code.google.com by `Eckankar` on 15 May 2011 at 10:24

**Comment by GoogleCodeExporter** [comment]
```
It's normal, there are beta features that could contains bugs.
You can have the description on the wiki page : 
http://code.google.com/p/gtalksms/wiki/BetaFeaturesOfGtalksms

Those features will be probably included in the next GTalkSMS version with full 
documentation.
```

Original comment by `Florent....@gmail.com` on 15 May 2011 at 10:52
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
Still a bug.

The "help:categories" command says:
- "help:system": battery, batt, ring, ringmode, exit, cmd,

This means you have a clash between a new command (system), and a command 
grouping in the help.

So, bullet point 1 is a legitimate bug.


```

Original comment by `Eckankar` on 15 May 2011 at 10:56

**Comment by GoogleCodeExporter** [comment]
```
Two options:
- rename "system" command
- rename "system" command group

I am in favor of the first.
```

Original comment by `fschm...@gmail.com` on 15 May 2011 at 3:35
- Changed title: **help:system clashes with system command group**
- Changed state: **Accepted**
- Added labels: **Priority-Low**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
System command renamed in sysinfo
```

Original comment by `Florent....@gmail.com` on 10 Jun 2011 at 10:30

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 10 Jun 2011 at 10:33
- Changed state: **WaitingForRelease**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v2.2.4
```

Original comment by `Florent....@gmail.com` on 13 Jun 2011 at 1:24
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 25 Aug 2011 at 7:50
- Changed state: **Deployed**
